src/NavBar/NavBar.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
+import SignIn from '../SignIn/SignIn';
 import './NavBar.css';
 
-const NavBar = function () {
+const NavBar = function (props) {
 	return (
 		<div className="nav-bar">
 			<a className="nav-bar__item">Submit a book</a>
 			<a className="nav-bar__item">Submissions</a>
 			<a className="nav-bar__item">Rate a book</a>
+			<SignIn isAuthenticated={props.isAuthenticated} />
```

Comment:
So here we're passing props.isAuthenticated into SignIn. 

Do components in react automatically get access to properties that parents pass in? It doesn't look like you declare that property anywhere in the SignIn, you just access it through props.

Reply:
Yeah, so parents pass props to children as attributes on elements (or at least, You can think of it like that syntactically). The Nav bar component reads in the props variable and then this component can pass those props to its children